rd_ui/app/views/dashboard.html

```diff
@@ -42,7 +42,7 @@ <h3 class="panel-title">
                     </h3>
                 </div>
 
-                <visualization-renderer visualization="widget.visualization" query-result="queryResult"></visualization-renderer class="panel-body">
+                <visualization-renderer visualization="widget.visualization" query-result="queryResult" hide-filters="dashboard.dashboard_filters_enabled"></visualization-renderer class="panel-body">
```

Comment:
The problem with this is that now you don't know if a widget is filtered or not. Also, sometimes the user might like to select different values on different widgets (that's actually less of a concern).